pkgs/applications/networking/browsers/badwolf/default.nix

```diff
@@ -1,39 +1,48 @@
-{ stdenv
-, lib
-, fetchgit
+{ lib
+, stdenv
+, fetchurl
+, ninja
 , pkg-config
+, ed
 , wrapGAppsHook
 , webkitgtk
 , libxml2
-, glib
 , glib-networking
 , gettext
 }:
+
 stdenv.mkDerivation rec {
   pname = "badwolf";
-  version = "1.2.2";
+  version = "1.3.0";
 
-  src = fetchgit {
-    url = "git://hacktivis.me/git/badwolf.git";
-    rev = "v${version}";
-    hash = "sha256-HfAsq6z+1kqMAsNxJjWJx9nd2cbv0XN4KRS8cYuhOsQ=";
+  src = fetchurl {
```

Comment:
Why switch from fetchgit to fetchurl?

Reply:
Because the original git repo doesn't work.

Reply:
Yes. What about `https://hacktivis.me/git/badwolf.git`?

Reply:
I'll try it.